ana_scans.py

```diff
@@ -217,13 +224,20 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
       print("Normal termination")
       pool.close()
       pool.join()
+      print "Results:", res.get()
+      for returncode in res.get():
+        if returncode != 0:
+          sys.exit(returncode)
+        pass
```

Comment:
Again could you describe the rationale here?  We're inside a `try` block already?

Also what are the possible `returncode` values?

Reply:
`returncode` is the return value from the called command, and will be non-zero if it fails (Python returns non-zero on exceptions, SIGSEGV, SIGKILL, ...). For this to work, we should ensure that our tools also return non-zero on "soft" failures (i.e. no crash), but that's a different issue.

Reply:
As above let's do this in a separate PR with a new issue